src/Tribe/Template/Month.php

```diff
@@ -603,7 +603,18 @@ protected function set_events_in_month() {
 			 */
 			$events_in_month = apply_filters( 'tribe_events_month_get_events_in_month', null, $start_date, $end_date );
 
+
 			if ( null === $events_in_month ) {
+				/**
+				 * Documented in the `\Tribe__Events__Query::pre_get_posts` method.
+				 *
+				 * @see   \Tribe__Events__Query::pre_get_posts()
+				 *
+				 * @since TBD
+				 */
+				$use_utc = apply_filters( 'tribe_events_query_event_utc_start_date', true );
```

Comment:
Pass the query here too. As the Second param.

Reply:
There is no query to pass here and passing the global one will mislead developers in thinking that query is the one that's being filtered.
I've improved the doc block on the `pre_get_posts` filter to add `null` as a second, possible, type for the `$query` parameter.